parquet-hadoop/src/test/java/org/apache/parquet/format/converter/TestParquetMetadataConverter.java

```diff
@@ -580,6 +583,42 @@ private void testBinaryStats(StatsHelper helper) {
     Assert.assertTrue(roundTripStats.isEmpty());
   }
 
+  @Test
+  public void testBinaryStatsWithTruncation() {
+    int defaultTruncLen = ParquetProperties.DEFAULT_STATISTICS_TRUNCATE_LENGTH;
+    int[] validLengths = {1, 2, 16, 64, defaultTruncLen - 1, defaultTruncLen - 1};
```

Comment:
Why do you test `defaultTruncLen -1` twice? I guess, `defaultTrundLen` itself shall also be tested.

Reply:
Good catch. That is a typo, I meant to test defaultTruncLen.

Reply:
`- 1` is still there. :)

Reply:
Don't know how it is slipped. Fix it now.